### PR TITLE
Feature/book trip

### PR DESCRIPTION
**Pull request by forsethnico**
What's this PR do?
- Add button and functionality to estimate cost. Also added button and book trip functionality (including the post to local API). 

How was this tested?
- Functionality was tested in the browser and the local API was updated with new trips. 

@letakeane

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -58,12 +70,17 @@ function getAllTravelData() {
...
-    currentTraveler = new Traveler(allTravelers[6]);
+    // currentTraveler = new Traveler(allTravelers[6]);
     setCurrentDate();
-    showDashboard();
   });
 }
 
+function onLoad() {
+    getAllTravelData().then(responses => {
+        showMainPage()
```
Comment (ohClaire) [inline src/scripts.js:80]: Add semicolon

**Review by ohClaire — CHANGES_REQUESTED** [review]
This looks great! There was one place missing a semicolon. I noticed that you are hidding several elements related to booking a trip so you are calling a hide function repetitively. I would recommend trying to DRY this up by putting all these elements in one container element so you only need to call hide once on that single html element.

File: src/scripts.js (hunk 1 above)
Comment (forsethnico) [reply src/scripts.js:80]: This has been added in a new branch.

**Comment by ohClaire** [comment]
This PR is for WIP functionality of the trip booking feature. The refactor will be done later to DRY this up.

**Pull request merged**